.gitlab-ci.yml

```diff
@@ -654,10 +654,32 @@ library:ci-argosy:
 library:ci-bbv:
   extends: .ci-template
 
+library:ci-coqutil:
+  extends: .ci-template-flambda
+
+library:ci-kami:
+  extends: .ci-template-flambda
+
+library:ci-coq_record_update:
+  extends: .ci-template-flambda
+
+library:ci-riscv_coq:
+  extends: .ci-template-flambda
+  stage: stage-3
+  needs:
+  - build:edge+flambda
+  - library:ci-coqutil
+
 library:ci-bedrock2:
   extends: .ci-template-flambda
   variables:
     NJOBS: "1"
+  stage: stage-4
+  needs:
+  - build:edge+flambda
+  - library:ci-coq_record_update
+  - library:ci-kami
+  - library:ci-riscv_coq
```

Comment:
You didn't revert these changes. I expect this is because you feel like it's useless to run bedrock2 if its dependencies have failed. But on the other hand, since dependencies will need to be rebuilt, this means that if there is a failure in bedrock2, it will take longer to learn about it.

Reply:
Perhaps it is better to keep this PR completely orthogonal to bedrock2's behaviour in the ci, and add the dependencies once the makefile has been fixed. Should I revert these changes also?

Reply:
Which changes? The dependency of bedrock2 on other libraries is not specified in `Makefile.ci` in the current version of the PR.

Reply:
@Zimmi48 Sorry for being unclear, I mean should I remove the dependencies in the gitlab job of bedrock2. The things we are commenting on.

In any case, it's probably not worth it any more till the makefile is fixed.